is_core/contrib/background_export/templates/is_core/generic_views/table.html

```diff
@@ -9,7 +9,7 @@
     <li>
         {% if export_types %}
         <div class="dropdown columns-dropdown">
-            <button class="btn btn-secondary dropdown-toggle" data-toggle="dropdown"><i class="fa fa-cloud-download"></i> {% trans 'Export' %}<span class="caret"></span></button>
+            <button class="btn btn-secondary dropdown-toggle" data-toggle="dropdown"><i class="fa fa-file-export"></i> <span class="caret"></span></button>
```

Comment:
Can you pls add tooltip for the icon-only buttons? Adding these attributes should be enough:
```
data-toggle="tooltip" title="Tooltip on bottom"
```